### PR TITLE
Removes headlines when all products are assessed

### PR DESCRIPTION
**Pull request by seijihg**
Without style adjustment:
Screenshot 2022-06-09 at 10 49 13

With style:
Screenshot 2022-06-08 at 17 07 17

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/LTD-2349
- [x] Jira ticket has the correct status.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: unit_tests/caseworker/tau/test_home.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def test_home_content(
 
     # Test elements of case info panel
     soup = BeautifulSoup(response.content, "html.parser")
-    assert soup.find(id="subtitle").text == "Assess 1 product going from Great Britain to Abu Dhabi and United Kingdom"
+    assert soup.find(id="subtitle").text == "Assess 1 product going from Great Britain to Abu Dhabi, United Kingdom"
```
Comment (saruniitr) [inline unit_tests/caseworker/tau/test_home.py:121]: this change is not related to this, I think if you pull `master` this will go away.

File: caseworker/assets/javascripts/tau-exporter-suggestions.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ export const addDeleteExporterCleSuggestions = (
 
   // See if product is checked if YES then create buttons.
   if (checked) {
-    globalCleMatchToItems[id].forEach((cle) => {
+    globalCleMatchToItems[id]?.forEach((cle) => {
```
Comment (saruniitr) [inline caseworker/assets/javascripts/tau-exporter-suggestions.js:117]: This is also not related. From what I know this `(using ?)` is not good as this is used to supress null checks?
Any particular reason why this needs to be added as it was not required before?

Comment (seijihg) [reply caseworker/assets/javascripts/tau-exporter-suggestions.js:117]: It is related to another thing maybe I should stop fixing not related issue to the ticket.

File: unit_tests/caseworker/tau/test_home.py (hunk 1 above)
Comment (seijihg) [reply unit_tests/caseworker/tau/test_home.py:121]: Because I rebased the master thats why test started failing.

File: caseworker/tau/templates/tau/home.html [hunk 3]
```diff
@@ -10,26 +10,21 @@
 
 <div class="govuk-width-container">
     <main class="govuk-main-wrapper tau">
-
-        <div class="govuk-grid-row">
+        <div class="govuk-grid-row tau__display--flex">
```
[the remark this answers is not in the document]
Comment (seijihg) [reply caseworker/tau/templates/tau/home.html:13]: It is still there? just added tau__display--flex

File: caseworker/assets/javascripts/tau-exporter-suggestions.js (hunk 2 above)
Comment (saruniitr) [reply caseworker/assets/javascripts/tau-exporter-suggestions.js:117]: yeah if it is not related to this ticket then separate PR is preferred.

I think a separate null check before this is better and return early if that is the case is better rather than this.

File: caseworker/tau/templates/tau/home.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (saruniitr) [reply caseworker/tau/templates/tau/home.html:13]: yes but is that really required?

File: unit_tests/caseworker/tau/test_home.py (hunk 1 above)
Comment (saruniitr) [reply unit_tests/caseworker/tau/test_home.py:121]: No, that was added to fix another ticket. This shouldn't fail if you have latest master.

If there are multiple countries we add `add` between the last two countries.

**Review by saruniitr — APPROVED** [review]
